nuxt-frontend/components/home-new/BlogSection.vue

```diff
@@ -16,7 +16,7 @@
           Well, at least the community says Hell Yeah. Our blogs hosted on
           medium have
           <span
-            class="secondary-color sub-h1-semibold lg:mobile-header-2-semibold"
+            class="v2-canopas-gradient-text sub-h1-semibold lg:mobile-header-2-semibold"
```

Comment:
Why changed color here?

Reply:
Its showing error of color contrast ratio on SEO. Its fine to use this